sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Configuration.java

```diff
@@ -76,8 +76,15 @@
     @Parameter(names = "-enableJvmStats", description = "Enables JVM Stats")
     private boolean enableJvmStats;
 
+    @Parameter(names = "-throughPutForMultiClientTest", description = "throughput on each container for multi client test")
+    private int throughPutForMultiClient = 100000;
+
+    @Parameter(names = "-deleteCollections", description = "Delete collections on all client with collectionId mentioned in the tool")
```

Comment:
it seems you are using `deleteCollections` for not only deleting collections but also for creating collection?

is that right? the config seems to be overloaded.

Reply:
during deleteCollections workload we don't want to create db/collection/data . createClient method during initialization will create client/db/container/data.

Reply:
two comments:
1. awesome, it is a good idea to automatically create and delete the collection, this will be useful for non multi client scenarios as well. Could you please extend to the non multi client scenario as well.
2. the description of the parameter and its name doesn't reflect what this is doing (how about `autoContainerCreationEnabled`

Reply:
discussed offline , new PR on benchmark will use autoCreateColection for all work flow